src/overlay/Peer.cpp

```diff
@@ -19,7 +19,7 @@
 #include "overlay/PeerRecord.h"
 #include "overlay/StellarXDR.h"
 #include "util/Logging.h"
-
+#include "util/XDROperators.h"
```

Comment:
please leave empty line after this, then clang-format wont be neccessary and library headers will be separated from inner ones